app/controllers/certificates_controller.rb

```diff
@@ -14,14 +14,15 @@ def create
     end
   end
 
-  def update
+  def enable
     @certificate = Certificate.find(params[:id])
+    EnableSigningCertificateEvent.create(certificate: @certificate)
```

Comment:
We should probably add some error handling here.

What should have when the state change fails and #create returns false ?